### PR TITLE
Task #5994 - Rspec model

### PR DESCRIPTION
**Pull request by taylq**
https://edu-redmine.framgia.vn/issues/5994?issue_count=19&issue_position=1&next_issue_id=5270

### CODE REVIEW

**Comment by taylq** [comment]
ready

File: app/models/author.rb [hunk 1]
```diff
@@ -1,3 +1,12 @@
...
-  has_many :book_authors, dependent: :destroy
+  has_many :authors_books, dependent: :destroy
+  has_many :books, through: :authors_books, dependent: :destroy
+
+  validates :name, presence: true
+
+  scope :select_attr, ->{select(:id, :name, :note)}
+
+  def author_ids_of_book book_id
+    authors_books.find_by(book_id: book_id)
```
Comment (huuphuc93) [inline app/models/author.rb]: check trường hợp k tìm thấy authors_books

**Comment by huuphuc93** [comment]
commented

**Comment by taylq** [comment]
ready

**Pull request merged**